exporter/clients_test.go

```diff
@@ -122,15 +117,15 @@ func TestExportClientList(t *testing.T) {
 
 		found := false
 		for m := range chM {
-			if strings.Contains(m.Desc().String(), "connected_clients_details") {
```

Comment:
I think you may want to revert some of these changes from `connected_clients_details` to `connected_client_info`

Reply:
added tests for both